<launch>
	<master auto="start"/>
	<param name="/use_sim_time" value="true"/>

	<include file="$(find p1)/move_base_config/move_base.xml"/>
	<node name="stageros" pkg="stage_ros" type="stageros" args="$(find p1)/stage_config/worlds/lab_home_5m.world" respawn="false" >
	<param name="base_watchdog_timeout" value="0.2"/>
	</node>

	<include file="$(find p1)/move_base_config/slam_gmapping_5m.xml"/>  
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find p1)/single_robot.rviz">
	</node>
	
	<node name="goal_list_client" pkg="p1" type="goal_list_client">
	</node>
	
	<node name="goal_list_server" pkg="p1" type="goal_list_server">
	</node>
	
	<node name="laser_server" pkg="p1" type="laser_server">
	</node>
  
</launch>
